README.md

```diff
@@ -39,8 +39,8 @@
 [Nodejs.org](https://nodejs.org/) by the [OpenJS Foundation](https://openjsf.org/) is the official website for the Node.js® JavaScript runtime. This repo is the source code for the website. It is built using [Next.js](https://nextjs.org), a React Framework.
 
 ```bash
-npm ci
-npm run dev
+pnpm i
```

Comment:
```suggestion
pnpm install --frozen-lockfile
```
I think we should use the long-hand `install` here for clarity, and we should use `--frozen-lockfile` to ensure dependencies match what's expected (and to match the old `npm ci` behaviour more closely)